src/chrome/content/rules/Wyndham.xml

```diff
@@ -126,26 +123,13 @@ Fetch error: http://www.wyndhamrentals.com/ => https://www.wyndhamrentals.com/:
 	<rule from="^http://(?:www\.)?(welcomecottages|resortquestsecure|secureholidays|worldmarkbywyndham|wyndhamrentals)\.com/"
 		to="https://www.$1.com/"/>
 
-	<rule from="^http://landal\.(\w[\w^z]\w?)/"
-		to="https://www.landal.$1/"/>
-
-	<!--	ditto	-->
-	<rule from="^http://www\.landal\.(\w[\w^z]\w?)/(bp|db|favicons|img)/"
-		to="https://www.landal.$1/$2/"/>
-
-	<rule from="^http://(?:www\.)?landal\.cz/"
-		to="https://www.landal.cz/"/>
-
 	<!--	webshop only for be & nl	-->
 	<rule from="^http://landal(campings|parkshop)\.([bd]e|nl)/"
 		to="https://www.landal$1.$2/"/>
 
 	<rule from="^http://www\.landacampings\.(be|nl)/(css|favicons|img)/"
 		to="https://www.landalcampings.$1/$2/"/>
 
-	<rule from="^http://secure\.landal\.com/"
-		to="https://secure.landal.com/"/>
-
 	<!--	redirects as so	-->
 	<rule from="^http://(?:www\.)?landalgreenparks\.com/"
 		to="https://www.landal.com/"/>
```

Comment:
What about that one?

Reply:
I'm sorry. Can you be more specific?

Reply:
The `landalgreenparks.com` hosts are redirected to `landal.com`. If you plan to keep this rule it should be in the `Landal.xml` file.